### PR TITLE
Add travis CI support

### PR DESCRIPTION
**Pull request by Rajik**
Added travis CI with a simpler travis.yml - and using travis-ci.com (https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps). We can change the README with your travis ci account once the PR is merged.

### CODE REVIEW

**Comment by till** [comment]
@Rajik Does it work with a standard Go build on Travis-CI? My guess is, we need docker support enabled cause we are building images etc. which compile all the tools, etc..

**Comment by till** [comment]
For reference: We setup CircleCI in our fork: https://github.com/pngmbh/workflow-cli/blob/master/.circleci/config.yml

**Comment by Rajik** [comment]
@till Yes the "make test" works without adding docker support explicitly in the travis.yml file. Please refer to the build logs [here](https://travis-ci.com/Rajik/workflow-cli). I will add the build step as well.

**Comment by Rajik** [comment]
I have added the build script to the stage.

**Comment by Cryptophobia** [comment]
Hi @Rajik , thank you for the PR and the contribution. 

What is the benefit of using travis-ci.com vs. travis-ci.org?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.com/Rajik/workflow-cli.svg?branch=master)](https://travis-ci.com/Rajik/workflow-cli)
```
Comment (Cryptophobia) [inline README.md]: Probably need to change this to `/teamhephy/workflow-cli`

**Comment by Rajik** [comment]
@Cryptophobia From the blog [here](https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps), I understand that travis-ci.com is the way forward for github apps and repos that are using travis-ci.org will have to migrate to travis-ci.com at a later point in time. So directly using travis-ci.com will save us some rework.

**Comment by Rajik** [comment]
Yes the builds are good on travis-ci.org for now. Shall I ignore that change for now?

**Comment by Cryptophobia** [comment]
Yes, please. I think just using travis-ci.org for now would be best.

**Comment by Rajik** [comment]
Made the changes and fixed something found in the process.

**Comment by Cryptophobia** [comment]
> Made the changes and fixed something found in the process.

Thanks @Rajik ! Looks good and build passes. Merging now.

**Pull request merged**